libyara/modules/pe.c

```diff
@@ -1386,6 +1386,10 @@ void pe_parse_header(
       OptionalHeader(Subsystem),
       pe->object, "subsystem");
 
+  set_integer(
+      OptionalHeader(DllCharacteristics),
+      pe->object, "dllcharacteristics");
```

Comment:
To be coherent with the naming schema of the module I would add an underscore: dll_characteristics